examples/ecs/event.rs

```diff
@@ -12,6 +12,8 @@ fn main() {
         .run();
 }
 
+#[derive(Event)]
+#[store(3)]
```

Comment:
I don't think `store(3)` is descriptive enough of what is happening here. We "store" all events, not just 3 of them. I think `#[smallvec(3)]` makes more sense. This has the added benefit of setting the pattern for future storage derives (ex: `#[vecdeque]`.

Reply:
Although im a little worried that we aren't properly scoping this attribute to events. Theres a world where another derive uses attribute names like `store` or `smallvec`.

Reply:
I *love* `#[smallvec(3)]` here for both of those reasons.

Reply:
`#[smallvec(3)]` is very hard to read without something linking the attribute to its derive

`#[event(storage = smallvec(3))]`